jib-core/src/main/java/com/google/cloud/tools/jib/http/ListenableCountingOutputStream.java

```diff
@@ -16,112 +16,71 @@
 
 package com.google.cloud.tools.jib.http;
 
-import com.google.common.annotations.VisibleForTesting;
 import java.io.IOException;
 import java.io.OutputStream;
-import java.time.Duration;
-import java.time.Instant;
 import java.util.function.Consumer;
-import java.util.function.Supplier;
 
-/**
- * Counts the number of bytes written and reports the count to a callback. The count is reported
- * with certain time delays to avoid calling the callback too often.
- */
+/** Counts the number of bytes written and reports the count to a callback. */
 public class ListenableCountingOutputStream extends OutputStream {
```

Comment:
Ugh this name.  `NotifyingOutputStream`?  Or even `CountingOutputStream`?

Reply:
Guava has `CountingOutputStream` (that just does byte counting, no listening), so going with `NotifyingOutputStream`.